Feature/school 15

riazonbin opened the pull request.

WingiM committed:
SCHOOL-12
Добавлен запрос на изменение имени учителя

riazonbin committed:
SCHOOL-15
Добавить страницу детального просмотра преподавателя с возможностью изменения личных данных (если роль - зам.диретора)

riazonbin committed:
SCHOOL-15
Убрать часть верстки, не связанной с изменением данных преподавателя

riazonbin committed:
hotfix/SCHOOL-15
Добавить диалоговое окно для подтверждения увольнения преподавателя; переход с главной страницы на страницу просмотра учитлей посредством навигационной панели

riazonbin committed:
Merge pull request #10 from Dumaem/hotfix/SCHOOL-15

hotfix/SCHOOL-15

WingiM commented on DumaemSchool.BlazorWeb/Pages/TeacherPage.razor:
```diff
@@ -0,0 +1,87 @@
+        await GetTeacherData();
+    }
+
+    private bool IsTeacherNameChange()
```
private bool TeacherNameChanged => currentTeacher?.Name != savedTeacherName

WingiM commented on DumaemSchool.BlazorWeb/Pages/TeachersPage.razor:
```diff
@@ -60,13 +63,34 @@
+
+        var result = await dialog.Result;
+
+        if (!result.Canceled)
```
return !result.Canceled

riazonbin committed:
SCHOOL-15
Изменения по PR: изменил метод проверки подтверждения увольнения учителя,  а также вместо метода использовал свойство для проверки изменения ФИО учителя

riazonbin committed:
SCHOOL-15
Влил в ветку dev для решения конфликтов
